packages/react-components/src/components/gauge/types.ts

```diff
@@ -24,6 +25,7 @@ export type GaugeBaseProperties = WidgetSettings &
     'thresholds' | 'settings' | 'significantDigits' | 'theme' | 'size'
   > & {
     isLoading?: boolean;
+    alarmState?: PascalCaseStateName;
```

Comment:
Let's include API BRs to review this change

Reply:
this is for `GaugeBaseProperties` so its an internal type update not a public API!

Reply:
Woops thought this was GaugeProps, all good!